content/videos/noc/9-genetic-algorithms/1-introduction/index.json

```diff
@@ -0,0 +1,55 @@
+{
+  "title": "9.1 Introduction",
+  "description": "Welcome to part 1 of a new series of videos focused on Evolutionary Computing, and more specifically, Genetic Algorithms. In this tutorial, I introduce the concept of a genetic algorithm, how it can be used to approach \"search\" problems and how it relates to brute force algorithms.",
+  "videoId": "9zfeTw-uFCw",
+  "date": "2016-07-29",
+  "languages": ["p5.js", "processing"],
+  "topics": ["genetic algorithm", "evolutionary computing"],
+  "canContribute": true,
+  "relatedChallenges": ["", ""],
```

Comment:
Many relatedChallenges here contain empty strings. Not sure if this will break, it would be better to keep an empty array instead.